src/engine/enginemaster.cpp

```diff
@@ -651,22 +656,24 @@ void EngineMaster::process(const int iBufferSize) {
             // mixing the talkover signal for the record/broadcast mix.
             // If not using microphone inputs or recording/broadcasting from
             // a sound card input, skip unnecessary processing here.
-            if (m_pNumMicsConfigured->get() > 0
-                && !m_bExternalRecordBroadcastInputConnected) {
+            if (m_pEngineSideChain &&
```

Comment:
This condition (the first 2 terms of the expression) appears 4 times. Please extract it as a method. It is complicated to read due to the dependencies that are not obvious at first sight.